packages/medusa/src/services/price-list.ts

```diff
@@ -239,7 +239,13 @@ class PriceListService extends BaseService {
     const priceListRepo = this.manager_.getCustomRepository(this.priceListRepo_)
```

Comment:
since this method is part of the public API provided by the service, it should be wrapped in an `atomicPhase_`